src/index.js

```diff
@@ -25,9 +25,15 @@
 const Routes = require('./routes')
 const Setup = require('.' +
     '/shared/setup')
+process.env.UV_THREADPOOL_SIZE = 12
```

Comment:
Please remove `process.env.UV_THREADPOOL_SIZE = 12`. <-- this can be configured via env variables as required.